.github/workflows/opentelemetry.yml

```diff
@@ -0,0 +1,19 @@
+---
+name: OpenTelemetry Export Trace
```

Comment:
Could you please clarify how OpenTelemetry is related to this PR?

Reply:
That's only to help us by visualisating the GitHub action executions as traces/spans. It's harmless but basically all the steps/jobs in gitHub actions will be reflects as span in one of our Elastic clusters. A similar approach was done with Jenkins, see https://github.com/elastic/ecs-logging-php/pull/43#issuecomment-1422847441 that contains a badge to the view in traces -> https://ci-stats.elastic.co/app/apm/services/apm-ci/transactions/view?rangeFrom=2023-02-08T15:43:48.299Z&rangeTo=2023-02-08T16:03:48.299Z&transactionName=BUILD+apm-agent-php%2Fecs-logging-php-mbp%2FPR-%7Bnumber%7D&transactionType=job&latencyAggregationType=avg&traceId=2051acf55017feda3c6a06fea1a278e4&transactionId=948a79d437b770ae